src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass_LocalFunctions.cs

```diff
@@ -58,6 +58,12 @@ protected TLocalFunctionState GetOrCreateLocalFuncUsages(LocalFunctionSymbol loc
 
         public override BoundNode? VisitLocalFunctionStatement(BoundLocalFunctionStatement localFunc)
         {
+            if (localFunc.Symbol.IsExtern)
```

Comment:
>localFunc.Symbol.IsExtern [](start = 16, length = 25)

Does this match the criteria used to skip analysis for type members (non-local functions)? Would it make sense to check for presence of the body instead?

Reply:
I found that for member methods, we decide whether to analyze based on whether BindMethodBody returned a non-null body. `extern` member methods will force a `null` return from BindMethodBody, which makes us skip analysis even in an error scenario where an extern method has a body.

I didn't want to modify the conditions when a BoundLocalFunctionStatement is given a `null` body, in order to reduce churn, and felt that this check resulted in useful diagnostic behaviors (i.e. if the function is required to have a body but one is missing, we might as well also let you know about things that are required to be included in that body), which are consistent enough with the diagnostics given for member methods.